WISE-Community/WISE
# Make sure the Authoring Tool logs out when the user session has timed out

Test sign out
1. Log in as a teacher
1. Open a project in the Authoring Tool
1. Open a new tab and go to the wise home page. Sign out.
1. Go back to the tab with the Authoring Tool and make a change to the project so that it auto saves
1. You should be redirected to login page

Test edit permission
1. Log in as a teacher
1. Open a project in the Authoring Tool that you do not have editing permission for
1. The upper right message should notify you that ```You do not have permission to edit this project```
1. In an incognito tab, log in as a teacher that does have editing permission for the project
1. Open that same project in the Authoring Tool and make sure the other teacher (that does not have editing permission) does not show up as a teacher "currently editing" the project.

Run the mvn tests and make sure they pass.
Run the wise5 tests and make sure they pass.
Run the site tests and make sure they pass.

Closes #2182
